REFERENCE.md

```diff
@@ -220,6 +220,8 @@ installed.
 
   * ipvs: The ability to match IP Virtual Server packets.
 
+  * ct_target: The ability to set connection tracking parameters for a packet or its associated connection.
```

Comment:
We can remove the updates to the REFERENCE.md file.
Puppet stringify feature will add the updates to the file.

Reply:
I've used puppet stringify to update the file, but if you want I can revert the change.

Reply:
Thats great.Thank you.